lib/schemas/api_v1.json

```diff
@@ -105,9 +105,11 @@
           "interactions": {
             "type": "object",
             "properties" : {
-              "likes" : { "$ref": "https://diaspora.software/api/v1/schema.json#/definitions/likes"},
-              "likes_count" : { "type": "integer"}
-            }
+              "liked" : { "type": "boolean" },
+              "likes_count" : { "type": "integer" }
```

Comment:
We show the people who liked a comment in the frontend. Is there a reason not to show a list of likees in the API as well?

Reply:
I think the reason is performance (as there could be quite many likes), there is a separate endpoint to get the complete list with all the information about the people who liked it. I assume fetching all the details about the people who liked it would just be too much when just fetching a list of posts/comments.

Reply:
> there is a separate endpoint to get the complete list with all the information about the people who liked it.

Okay, I have to admit, I 100% missed this. This is perfect, then, I'll mark this as resolved.